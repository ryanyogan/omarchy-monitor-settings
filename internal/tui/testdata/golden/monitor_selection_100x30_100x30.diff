Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m│[0m                                      [1;34mDisplay Settings[0m                                      [90m│[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                    
  [34m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m  [1;34mMonitor Selection[0m                                                                         [34m│[0m    
  [34m│[0m  [37mChoose a display to configure[0m                                                             [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m  [1;32m▶ [0m[1;32mHDMI-A-1[0m [44m [0m[1;44mACTIVE[0m[44m [0m                                                                       [34m│[0m    
  [34m│[0m    [37mDell U2414H[0m                                                                             [34m│[0m    
  [34m│[0m    [37m1920x1080 @ 60Hz[0m                                                                        [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m    [1;34mDP-1[0m [100m [0m[1;100mINACTIVE[0m[100m [0m                                                                         [34m│[0m    
  [34m│[0m      [37mSamsung C27F390[0m                                                                       [34m│[0m    
  [34m│[0m      [37m1920x1080 @ 75Hz[0m                                                                      [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m  [33m⏎[0m[37m Select monitor and return to dashboard[0m  [35mesc[0m[37m Return to main menu[0m                         [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m  [3;90m💡 Selected monitor will be marked as CURRENT on the dashboard[0m                            [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m│[0m                                                                                            [34m│[0m    
  [34m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                    
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m│[0m                 [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m               [90m│[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                      Display Settings                                      │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                                                                            │    
  │  Monitor Selection                                                                         │    
  │  Choose a display to configure                                                             │    
  │                                                                                            │    
  │  ▶ HDMI-A-1  ACTIVE                                                                        │    
  │    Dell U2414H                                                                             │    
  │    1920x1080 @ 60Hz                                                                        │    
  │                                                                                            │    
  │    DP-1  INACTIVE                                                                          │    
  │      Samsung C27F390                                                                       │    
  │      1920x1080 @ 75Hz                                                                      │    
  │                                                                                            │    
  │                                                                                            │    
  │  ⏎ Select monitor and return to dashboard  esc Return to main menu                         │    
  │                                                                                            │    
  │  💡 Selected monitor will be marked as CURRENT on the dashboard                            │    
  │                                                                                            │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                 ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit               │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 5384
Actual length: 4572
Expected hash: 01e2bd38071c782719002691e422a715842fc6652251ca27dfc0906a60cfed22
Actual hash: c8c532e3543975defc4413ccc51207f1a6e8fff81f57258922bd2781cb8a79b2
