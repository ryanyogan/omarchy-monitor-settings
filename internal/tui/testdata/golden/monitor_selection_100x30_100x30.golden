# Visual Golden File
# Name: monitor_selection_100x30
# Dimensions: 100x30
# Hash: c8c532e3543975defc4413ccc51207f1a6e8fff81f57258922bd2781cb8a79b2

  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                      Display Settings                                      │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                                                                            │    
  │  Monitor Selection                                                                         │    
  │  Choose a display to configure                                                             │    
  │                                                                                            │    
  │  ▶ HDMI-A-1  ACTIVE                                                                        │    
  │    Dell U2414H                                                                             │    
  │    1920x1080 @ 60Hz                                                                        │    
  │                                                                                            │    
  │    DP-1  INACTIVE                                                                          │    
  │      Samsung C27F390                                                                       │    
  │      1920x1080 @ 75Hz                                                                      │    
  │                                                                                            │    
  │                                                                                            │    
  │  ⏎ Select monitor and return to dashboard  esc Return to main menu                         │    
  │                                                                                            │    
  │  💡 Selected monitor will be marked as CURRENT on the dashboard                            │    
  │                                                                                            │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                 ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit               │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    